src/main/scala/firrtl/passes/clocklist/ClockListUtils.scala

```diff
@@ -48,12 +48,14 @@ object ClockListUtils {
       case Module(i, n, ports, b) => ports.collectFirst { case p if p.name == "clock" => me + sep + "clock" }
       case ExtModule(i, n, ports, dn, p) => None
     }
-    // Return new origins with children removed, if they match my clock
+    // Return new origins with direct children removed, if they match my clock
     clockOpt match {
       case Some(clock) =>
         val myOrigin = getOrigin(connects, clock).serialize
         childrenOrigins.foldLeft(Map(me -> myOrigin)) { case (o, (childInstance, childOrigin)) =>
-          if(childOrigin == myOrigin) o else o + (childInstance -> childOrigin)
+          val childrenInstances = lin.children.map { case (i, l) => me + sep + i }
```

Comment:
Singleton "l" is a ugly variable name, and since it isn't being used shouldn't it just be an underscore